setup.py

```diff
@@ -342,8 +341,7 @@ def run(self):
 
```

Comment:
I can't comment on the line, but should we remove `opentimelineio_contrib` from `package_data`?

Reply:
Hmm, I must have missed that one.